docs/source/tutorials/tutorial.ipynb

```diff
@@ -72,23 +72,10 @@
     "This notebook uses an example dataset, that you can download using these commands."
    ]
   },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "wget - nc 'https://cleanlab-public.s3.amazonaws.com/CleanVision/image_files.zip'\n",
-    "\n",
-    "unzip -q image_files.zip"
-   ]
-  },
```

Comment:
I did not understand why there was these commands in the markdown cell and then again in hidden code cell. These commands did not look good in the markdown cell on our website:

![Screen Shot 2023-11-16 at 7 58 52 PM](https://github.com/cleanlab/cleanvision/assets/1390638/b59c1a8f-9250-4155-ac74-d53ac6b1fb6c)

Reply:
![Screenshot 2023-11-17 at 9 32 38 AM](https://github.com/cleanlab/cleanvision/assets/7639432/9749d37c-ab2a-4bf7-b229-74117028cb4d)
I put it in a hidden cell to hide this output.

Reply:
There's no tag in nbsphinx that hides just the output, it's either whole cell or nothing

Reply:
oh then can you make it look like code in the markdown cell? on the current website, this just doesn't look like code, which is suboptimal formatting.  I agree hiding the output is good

Reply:
at the very top of the notebook in the hidden cell that'll be visible in colab that installs cleanvision from master branch, can you also put a comment:

`pip install cleanvision` to use the stable release of the package instead.

Reply:
okay